lib/types/validation-error.ts

```diff
@@ -76,6 +76,15 @@ export interface TypeValidationError extends BaseValidationError {
   type: string
 }
 
+export interface AllowedResourceValidationError extends BaseValidationError {
+  name: 'allowedResource'
+  code: string
+}
+export interface AllowedResourcesValidationError extends BaseValidationError {
+  name: 'allowedResources'
+  contentTypeId?: string[]
```

Comment:
nitpick: 

```suggestion
  contentTypeIds?: string[]
```

Reply:
It is `contentTypeId` though 😁
https://github.com/contentful/content-stack/blob/master/packages/validation/test/unit/resource-link-allowed-resource-test.js#L154